### :x: 2 Tests Failed:
| Tests completed | Failed | Passed | Skipped |
|---|---|---|---|
| 2747 | 2 | 2745 | 30 |
<details><summary>View the top 2 failed tests by shortest run time</summary>

> 
> ```
> github.com/status-im/status-go/protocol TestMessengerSendImagesAlbumSuite/TestAlbumImageEditText
> ```
> 
> <details><summary>Stack Traces | 1.66s run time</summary>
> 
> > 
> > ```
> > === RUN   TestMessengerSendImagesAlbumSuite/TestAlbumImageEditText&#xA;2024-09-10T16:47:47.997Z&#x9;DEBUG&#x9;waku/waku.go:126&#x9;starting waku with config&#x9;{&#34;config&#34;: {&#34;MaxMessageSize&#34;:1048576,&#34;MinimumAcceptedPoW&#34;:0,&#34;BloomFilterMode&#34;:false,&#34;LightClient&#34;:false,&#34;FullNode&#34;:false,&#34;RestrictLightClientsConn&#34;:true,&#34;EnableConfirmations&#34;:false,&#34;SoftBlacklistedPeerIDs&#34;:null}}&#xA;2024-09-10T16:47:48.000Z&#x9;DEBUG&#x9;messenger-Alice&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0404ab81&#34;, &#34;epoch&#34;: 846}}&#xA;2024-09-10T16:47:48.000Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;041caacc&#34;, &#34;epoch&#34;: 854}}&#xA;2024-09-10T16:47:48.000Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;049ee811&#34;, &#34;epoch&#34;: 1557}}&#xA;2024-09-10T16:47:48.012Z&#x9;ERROR&#x9;node/node.go:386&#x9;error while mapping sync state&#x9;{&#34;mvds&#34;: {&#34;error&#34;: &#34;sql: database is closed&#34;}}&#xA;github..../status-im/mvds/node.(*Node).sendMessages&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../mvds/node/node.go:386&#xA;github..../status-im/mvds/node.(*Node).Start.func3&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../mvds/node/node.go:223&#xA;2024-09-10T16:47:48.012Z&#x9;ERROR&#x9;node/node.go:225&#x9;Error sending messages.&#x9;{&#34;mvds&#34;: {&#34;error&#34;: &#34;sql: database is closed&#34;}}&#xA;github..../status-im/mvds/node.(*Node).Start.func3&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../mvds/node/node.go:225&#xA;2024-09-10T16:47:48.012Z&#x9;ERROR&#x9;node/node.go:229&#x9;Error clearing sync state.&#x9;{&#34;mvds&#34;: {&#34;error&#34;: &#34;sql: database is closed&#34;}}&#xA;github..../status-im/mvds/node.(*Node).Start.func3&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../mvds/node/node.go:229&#xA;2024-09-10T16:47:48.012Z&#x9;ERROR&#x9;node/node.go:235&#x9;Failed to persisten epoch&#x9;{&#34;mvds&#34;: {&#34;error&#34;: &#34;sql: database is closed&#34;}}&#xA;github..../status-im/mvds/node.(*Node).Start.func3&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../mvds/node/node.go:235&#xA;2024-09-10T16:47:48.012Z&#x9;INFO&#x9;node/node.go:218&#x9;Epoch processing stopped&#x9;{&#34;mvds&#34;: {}}&#xA;2024-09-10T16:47:48.054Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04485dfe&#34;, &#34;epoch&#34;: 1951}}&#xA;2024-09-10T16:47:48.054Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;048b92e4&#34;, &#34;epoch&#34;: 2220}}&#xA;2024-09-10T16:47:48.071Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04ad37f5&#34;, &#34;epoch&#34;: 1930}}&#xA;2024-09-10T16:47:48.071Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040b3c22&#34;, &#34;epoch&#34;: 1554}}&#xA;WARN [09-10|16:47:48.073|github..../status-go/appdatabase/database.go:137]                       Migrating accounts: no wallet_root_address found in settings &#xA;2024-09-10T16:47:48.074Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040cd6d3&#34;, &#34;epoch&#34;: 1547}}&#xA;2024-09-10T16:47:48.138Z&#x9;DEBUG&#x9;messenger-Bobby&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040a5dd5&#34;, &#34;epoch&#34;: 845}}&#xA;2024-09-10T16:47:48.220Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0458c277&#34;, &#34;epoch&#34;: 2201}}&#xA;2024-09-10T16:47:48.220Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04189afe&#34;, &#34;epoch&#34;: 2167}}&#xA;2024-09-10T16:47:48.220Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04420f87&#34;, &#34;epoch&#34;: 2211}}&#xA;2024-09-10T16:47:48.220Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04f955ef&#34;, &#34;epoch&#34;: 2191}}&#xA;2024-09-10T16:47:48.220Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0489298c&#34;, &#34;epoch&#34;: 1073}}&#xA;2024-09-10T16:47:48.230Z&#x9;ERROR&#x9;node/node.go:386&#x9;error while mapping sync state&#x9;{&#34;mvds&#34;: {&#34;error&#34;: &#34;sql: database is closed&#34;}}&#xA;github..../status-im/mvds/node.(*Node).sendMessages&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../mvds/node/node.go:386&#xA;github..../status-im/mvds/node.(*Node).Start.func3&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../mvds/node/node.go:223&#xA;2024-09-10T16:47:48.230Z&#x9;ERROR&#x9;node/node.go:225&#x9;Error sending messages.&#x9;{&#34;mvds&#34;: {&#34;error&#34;: &#34;sql: database is closed&#34;}}&#xA;github..../status-im/mvds/node.(*Node).Start.func3&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../mvds/node/node.go:225&#xA;2024-09-10T16:47:48.230Z&#x9;ERROR&#x9;node/node.go:229&#x9;Error clearing sync state.&#x9;{&#34;mvds&#34;: {&#34;error&#34;: &#34;sql: database is closed&#34;}}&#xA;github..../status-im/mvds/node.(*Node).Start.func3&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../mvds/node/node.go:229&#xA;2024-09-10T16:47:48.230Z&#x9;ERROR&#x9;node/node.go:235&#x9;Failed to persisten epoch&#x9;{&#34;mvds&#34;: {&#34;error&#34;: &#34;sql: database is closed&#34;}}&#xA;github..../status-im/mvds/node.(*Node).Start.func3&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../mvds/node/node.go:235&#xA;2024-09-10T16:47:48.230Z&#x9;INFO&#x9;node/node.go:218&#x9;Epoch processing stopped&#x9;{&#34;mvds&#34;: {}}&#xA;2024-09-10T16:47:48.242Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;045add39&#34;, &#34;epoch&#34;: 269}}&#xA;2024-09-10T16:47:48.257Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04a90e8f&#34;, &#34;epoch&#34;: 1551}}&#xA;2024-09-10T16:47:48.300Z&#x9;DEBUG&#x9;messenger-Alice&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0404ab81&#34;, &#34;epoch&#34;: 847}}&#xA;2024-09-10T16:47:48.300Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;049ee811&#34;, &#34;epoch&#34;: 1558}}&#xA;2024-09-10T16:47:48.300Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;041caacc&#34;, &#34;epoch&#34;: 855}}&#xA;2024-09-10T16:47:48.355Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04485dfe&#34;, &#34;epoch&#34;: 1952}}&#xA;2024-09-10T16:47:48.355Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;048b92e4&#34;, &#34;epoch&#34;: 2221}}&#xA;2024-09-10T16:47:48.372Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040b3c22&#34;, &#34;epoch&#34;: 1555}}&#xA;2024-09-10T16:47:48.372Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04ad37f5&#34;, &#34;epoch&#34;: 1931}}&#xA;2024-09-10T16:47:48.375Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040cd6d3&#34;, &#34;epoch&#34;: 1548}}&#xA;2024-09-10T16:47:48.396Z&#x9;DEBUG&#x9;protocol/messenger_status_updates.go:104&#x9;Error cleaning status updates&#x9;{&#34;error&#34;: &#34;sql: database is closed&#34;}&#xA;2024-09-10T16:47:48.439Z&#x9;DEBUG&#x9;messenger-Bobby&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040a5dd5&#34;, &#34;epoch&#34;: 846}}&#xA;2024-09-10T16:47:48.520Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04189afe&#34;, &#34;epoch&#34;: 2168}}&#xA;2024-09-10T16:47:48.521Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0489298c&#34;, &#34;epoch&#34;: 1074}}&#xA;2024-09-10T16:47:48.521Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04420f87&#34;, &#34;epoch&#34;: 2212}}&#xA;2024-09-10T16:47:48.521Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0458c277&#34;, &#34;epoch&#34;: 2202}}&#xA;2024-09-10T16:47:48.521Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04f955ef&#34;, &#34;epoch&#34;: 2192}}&#xA;2024-09-10T16:47:48.542Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;045add39&#34;, &#34;epoch&#34;: 270}}&#xA;2024-09-10T16:47:48.558Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04a90e8f&#34;, &#34;epoch&#34;: 1552}}&#xA;2024-09-10T16:47:48.601Z&#x9;DEBUG&#x9;messenger-Alice&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0404ab81&#34;, &#34;epoch&#34;: 848}}&#xA;2024-09-10T16:47:48.601Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;041caacc&#34;, &#34;epoch&#34;: 856}}&#xA;2024-09-10T16:47:48.601Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;049ee811&#34;, &#34;epoch&#34;: 1559}}&#xA;2024-09-10T16:47:48.606Z&#x9;DEBUG&#x9;transport/filters_manager.go:612&#x9;registering filter for&#x9;{&#34;filtersManager&#34;: {&#34;chatID&#34;: &#34;0x0468d3e5684b73cabf155749ce52aa9e788945bb26f98dfbd9a1d38bdf8217074ea0a2b16a4de4da4549346f3ba9544863deae1b82cfb18adf49a2946f7d8cba53-contact-code&#34;, &#34;type&#34;: &#34;contact-code&#34;, &#34;topic&#34;: &#34;0x2b30fa7a&#34;}}&#xA;2024-09-10T16:47:48.607Z&#x9;DEBUG&#x9;transport/filters_manager.go:441&#x9;registering filter for&#x9;{&#34;filtersManager&#34;: {&#34;chatID&#34;: &#34;contact-discovery-3534&#34;, &#34;type&#34;: &#34;partitioned&#34;, &#34;topic&#34;: &#34;0x1fed6e76&#34;}}&#xA;2024-09-10T16:47:48.608Z&#x9;DEBUG&#x9;transport/filters_manager.go:525&#x9;registering filter for&#x9;{&#34;filtersManager&#34;: {&#34;chatID&#34;: &#34;contact-discovery-0468d3e5684b73cabf155749ce52aa9e788945bb26f98dfbd9a1d38bdf8217074ea0a2b16a4de4da4549346f3ba9544863deae1b82cfb18adf49a2946f7d8cba53&#34;, &#34;type&#34;: &#34;discovery&#34;, &#34;topic&#34;: &#34;0xb69c5e00&#34;}}&#xA;2024-09-10T16:47:48.608Z&#x9;INFO&#x9;protocol/messenger.go:793&#x9;starting messenger&#x9;{&#34;identity&#34;: &#34;0x0468d3e5684b73cabf155749ce52aa9e788945bb26f98dfbd9a1d38bdf8217074ea0a2b16a4de4da4549346f3ba9544863deae1b82cfb18adf49a2946f7d8cba53&#34;}&#xA;2024-09-10T16:47:48.608Z&#x9;INFO&#x9;communities/manager.go:616&#x9;starting owner verification loop&#xA;2024-09-10T16:47:48.608Z&#x9;INFO&#x9;publisher/publisher.go:48&#x9;starting publisher&#x9;{&#34;Publisher&#34;: {&#34;site&#34;: &#34;Start&#34;}}&#xA;2024-09-10T16:47:48.608Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:713&#x9;subscribing for message events&#xA;2024-09-10T16:47:48.608Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1502&#x9;running resending loop&#xA;2024-09-10T16:47:48.608Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1561&#x9;running registration loop&#xA;2024-09-10T16:47:48.608Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0468d3e5&#34;, &#34;epoch&#34;: 0}}&#xA;2024-09-10T16:47:48.608Z&#x9;DEBUG&#x9;protocol/messenger.go:1583&#x9;watching connection changes&#xA;2024-09-10T16:47:48.608Z&#x9;DEBUG&#x9;protocol/messenger_mailserver_cycle.go:417&#x9;asyncRequestAllHistoricMessages&#xA;2024-09-10T16:47:48.608Z&#x9;WARN&#x9;protocol/messenger.go:1590&#x9;using WakuV1, can&#39;t watch connection changes, this might be have side-effects&#xA;github..../status-im/status-go/protocol.(*Messenger).watchConnectionChange&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger.go:1590&#xA;github..../status-im/status-go/protocol.(*Messenger).Start&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger.go:851&#xA;github..../status-im/status-go/protocol.(*MessengerSendImagesAlbumSuite).SetupTest&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger_send_images_album_test.go:28&#xA;github..../stretchr/testify/suite.Run.func1&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../testify/suite/suite.go:192&#xA;testing.tRunner&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/testing/testing.go:1595&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;protocol/messenger.go:1604&#x9;Checking for chats to unmute every minute&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;protocol/messenger.go:1647&#x9;Checking for communities to unmute every minute&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1502&#x9;running resending loop&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;protocol/messenger_raw_message_resend.go:19&#x9;watching expired messages&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;protocol/messenger.go:1675&#x9;watching identity image changes&#xA;2024-09-10T16:47:48.609Z&#x9;INFO&#x9;protocol/messenger_wallet.go:61&#x9;watching wallet balances&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;protocol/messenger_communities.go:838&#x9;watching communities to unmute&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1561&#x9;running registration loop&#xA;2024-09-10T16:47:48.609Z&#x9;WARN&#x9;protocol/messenger_wallet.go:64&#x9;wallet service not enabled&#xA;github..../status-im/status-go/protocol.(*Messenger).watchWalletBalances&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger_wallet.go:64&#xA;github..../status-im/status-go/protocol.(*Messenger).Start&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger.go:856&#xA;github..../status-im/status-go/protocol.(*MessengerSendImagesAlbumSuite).SetupTest&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger_send_images_album_test.go:28&#xA;github..../stretchr/testify/suite.Run.func1&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../testify/suite/suite.go:192&#xA;testing.tRunner&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/testing/testing.go:1595&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;protocol/messenger.go:1717&#x9;watching community request to join&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;protocol/messenger_status_updates.go:194&#x9;broadcasting user status&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1513&#x9;no retriable notifications, quitting&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1513&#x9;no retriable notifications, quitting&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1568&#x9;nothing to do, quitting registration loop&#xA;2024-09-10T16:47:48.609Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1568&#x9;nothing to do, quitting registration loop&#xA;2024-09-10T16:47:48.611Z&#x9;WARN&#x9;protocol/messenger_mailserver_cycle.go:54&#x9;not starting mailserver cycle: requires wakuv2&#xA;github..../status-im/status-go/protocol.(*Messenger).StartMailserverCycle&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger_mailserver_cycle.go:54&#xA;github..../status-im/status-go/protocol.(*Messenger).Start&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger.go:891&#xA;github..../status-im/status-go/protocol.(*MessengerSendImagesAlbumSuite).SetupTest&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger_send_images_album_test.go:28&#xA;github..../stretchr/testify/suite.Run.func1&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../testify/suite/suite.go:192&#xA;testing.tRunner&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/testing/testing.go:1595&#xA;2024-09-10T16:47:48.611Z&#x9;DEBUG&#x9;storenodes/storenodes.go:98&#x9;loaded communities storenodes&#x9;{&#34;CommunityStorenodes&#34;: {&#34;count&#34;: 0}}&#xA;2024-09-10T16:47:48.655Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04485dfe&#34;, &#34;epoch&#34;: 1953}}&#xA;2024-09-10T16:47:48.655Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;048b92e4&#34;, &#34;epoch&#34;: 2222}}&#xA;2024-09-10T16:47:48.672Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040b3c22&#34;, &#34;epoch&#34;: 1556}}&#xA;2024-09-10T16:47:48.672Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04ad37f5&#34;, &#34;epoch&#34;: 1932}}&#xA;2024-09-10T16:47:48.676Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040cd6d3&#34;, &#34;epoch&#34;: 1549}}&#xA;WARN [09-10|16:47:48.687|github..../status-go/appdatabase/database.go:137]                       Migrating accounts: no wallet_root_address found in settings &#xA;2024-09-10T16:47:48.740Z&#x9;DEBUG&#x9;messenger-Bobby&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040a5dd5&#34;, &#34;epoch&#34;: 847}}&#xA;2024-09-10T16:47:48.821Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04189afe&#34;, &#34;epoch&#34;: 2169}}&#xA;2024-09-10T16:47:48.822Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04420f87&#34;, &#34;epoch&#34;: 2213}}&#xA;2024-09-10T16:47:48.822Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0489298c&#34;, &#34;epoch&#34;: 1075}}&#xA;2024-09-10T16:47:48.822Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04f955ef&#34;, &#34;epoch&#34;: 2193}}&#xA;2024-09-10T16:47:48.822Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0458c277&#34;, &#34;epoch&#34;: 2203}}&#xA;2024-09-10T16:47:48.842Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;045add39&#34;, &#34;epoch&#34;: 271}}&#xA;2024-09-10T16:47:48.858Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04a90e8f&#34;, &#34;epoch&#34;: 1553}}&#xA;2024-09-10T16:47:48.902Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;041caacc&#34;, &#34;epoch&#34;: 857}}&#xA;2024-09-10T16:47:48.902Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;049ee811&#34;, &#34;epoch&#34;: 1560}}&#xA;2024-09-10T16:47:48.902Z&#x9;DEBUG&#x9;messenger-Alice&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0404ab81&#34;, &#34;epoch&#34;: 849}}&#xA;2024-09-10T16:47:48.909Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0468d3e5&#34;, &#34;epoch&#34;: 1}}&#xA;2024-09-10T16:47:48.955Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04485dfe&#34;, &#34;epoch&#34;: 1954}}&#xA;2024-09-10T16:47:48.957Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;048b92e4&#34;, &#34;epoch&#34;: 2223}}&#xA;2024-09-10T16:47:48.972Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040b3c22&#34;, &#34;epoch&#34;: 1557}}&#xA;2024-09-10T16:47:48.972Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04ad37f5&#34;, &#34;epoch&#34;: 1933}}&#xA;2024-09-10T16:47:48.976Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040cd6d3&#34;, &#34;epoch&#34;: 1550}}&#xA;2024-09-10T16:47:49.040Z&#x9;DEBUG&#x9;messenger-Bobby&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040a5dd5&#34;, &#34;epoch&#34;: 848}}&#xA;2024-09-10T16:47:49.122Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04f955ef&#34;, &#34;epoch&#34;: 2194}}&#xA;2024-09-10T16:47:49.122Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0489298c&#34;, &#34;epoch&#34;: 1076}}&#xA;2024-09-10T16:47:49.122Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04420f87&#34;, &#34;epoch&#34;: 2214}}&#xA;2024-09-10T16:47:49.122Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04189afe&#34;, &#34;epoch&#34;: 2170}}&#xA;2024-09-10T16:47:49.122Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0458c277&#34;, &#34;epoch&#34;: 2204}}&#xA;2024-09-10T16:47:49.144Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;045add39&#34;, &#34;epoch&#34;: 272}}&#xA;2024-09-10T16:47:49.158Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04a90e8f&#34;, &#34;epoch&#34;: 1554}}&#xA;2024-09-10T16:47:49.174Z&#x9;DEBUG&#x9;transport/filters_manager.go:612&#x9;registering filter for&#x9;{&#34;filtersManager&#34;: {&#34;chatID&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83-contact-code&#34;, &#34;type&#34;: &#34;contact-code&#34;, &#34;topic&#34;: &#34;0xa7171021&#34;}}&#xA;2024-09-10T16:47:49.174Z&#x9;DEBUG&#x9;transport/filters_manager.go:441&#x9;registering filter for&#x9;{&#34;filtersManager&#34;: {&#34;chatID&#34;: &#34;contact-discovery-872&#34;, &#34;type&#34;: &#34;partitioned&#34;, &#34;topic&#34;: &#34;0x19c723b4&#34;}}&#xA;2024-09-10T16:47:49.175Z&#x9;DEBUG&#x9;transport/filters_manager.go:525&#x9;registering filter for&#x9;{&#34;filtersManager&#34;: {&#34;chatID&#34;: &#34;contact-discovery-04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;, &#34;type&#34;: &#34;discovery&#34;, &#34;topic&#34;: &#34;0x2b655306&#34;}}&#xA;2024-09-10T16:47:49.176Z&#x9;INFO&#x9;protocol/messenger.go:793&#x9;starting messenger&#x9;{&#34;identity&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;}&#xA;2024-09-10T16:47:49.176Z&#x9;INFO&#x9;communities/manager.go:616&#x9;starting owner verification loop&#xA;2024-09-10T16:47:49.176Z&#x9;INFO&#x9;publisher/publisher.go:48&#x9;starting publisher&#x9;{&#34;Publisher&#34;: {&#34;site&#34;: &#34;Start&#34;}}&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:713&#x9;subscribing for message events&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1502&#x9;running resending loop&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1561&#x9;running registration loop&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04950fc7&#34;, &#34;epoch&#34;: 0}}&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;protocol/messenger.go:1583&#x9;watching connection changes&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;protocol/messenger_mailserver_cycle.go:417&#x9;asyncRequestAllHistoricMessages&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1513&#x9;no retriable notifications, quitting&#xA;2024-09-10T16:47:49.176Z&#x9;WARN&#x9;protocol/messenger.go:1590&#x9;using WakuV1, can&#39;t watch connection changes, this might be have side-effects&#xA;github..../status-im/status-go/protocol.(*Messenger).watchConnectionChange&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger.go:1590&#xA;github..../status-im/status-go/protocol.(*Messenger).Start&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger.go:851&#xA;github..../status-im/status-go/protocol.(*MessengerSendImagesAlbumSuite).TestAlbumImageEditText&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger_send_images_album_test.go:212&#xA;reflect.Value.call&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/reflect/value.go:596&#xA;reflect.Value.Call&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/reflect/value.go:380&#xA;github..../stretchr/testify/suite.Run.func1&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../testify/suite/suite.go:202&#xA;testing.tRunner&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/testing/testing.go:1595&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;protocol/messenger.go:1604&#x9;Checking for chats to unmute every minute&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1561&#x9;running registration loop&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;protocol/messenger.go:1647&#x9;Checking for communities to unmute every minute&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1568&#x9;nothing to do, quitting registration loop&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;protocol/messenger_raw_message_resend.go:19&#x9;watching expired messages&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1502&#x9;running resending loop&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;protocol/messenger.go:1675&#x9;watching identity image changes&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;protocol/messenger_communities.go:838&#x9;watching communities to unmute&#xA;2024-09-10T16:47:49.176Z&#x9;INFO&#x9;protocol/messenger_wallet.go:61&#x9;watching wallet balances&#xA;2024-09-10T16:47:49.176Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1568&#x9;nothing to do, quitting registration loop&#xA;2024-09-10T16:47:49.176Z&#x9;WARN&#x9;protocol/messenger_wallet.go:64&#x9;wallet service not enabled&#xA;github..../status-im/status-go/protocol.(*Messenger).watchWalletBalances&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger_wallet.go:64&#xA;github..../status-im/status-go/protocol.(*Messenger).Start&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger.go:856&#xA;github..../status-im/status-go/protocol.(*MessengerSendImagesAlbumSuite).TestAlbumImageEditText&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger_send_images_album_test.go:212&#xA;reflect.Value.call&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/reflect/value.go:596&#xA;reflect.Value.Call&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/reflect/value.go:380&#xA;github..../stretchr/testify/suite.Run.func1&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../testify/suite/suite.go:202&#xA;testing.tRunner&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/testing/testing.go:1595&#xA;2024-09-10T16:47:49.177Z&#x9;DEBUG&#x9;protocol/messenger.go:1717&#x9;watching community request to join&#xA;2024-09-10T16:47:49.177Z&#x9;DEBUG&#x9;protocol/messenger_status_updates.go:194&#x9;broadcasting user status&#xA;2024-09-10T16:47:49.177Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:1513&#x9;no retriable notifications, quitting&#xA;2024-09-10T16:47:49.179Z&#x9;WARN&#x9;protocol/messenger_mailserver_cycle.go:54&#x9;not starting mailserver cycle: requires wakuv2&#xA;github..../status-im/status-go/protocol.(*Messenger).StartMailserverCycle&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger_mailserver_cycle.go:54&#xA;github..../status-im/status-go/protocol.(*Messenger).Start&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger.go:891&#xA;github..../status-im/status-go/protocol.(*MessengerSendImagesAlbumSuite).TestAlbumImageEditText&#xA;&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger_send_images_album_test.go:212&#xA;reflect.Value.call&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/reflect/value.go:596&#xA;reflect.Value.Call&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/reflect/value.go:380&#xA;github..../stretchr/testify/suite.Run.func1&#xA;&#x9;.../status-go_prs_tests_PR-5823/vendor/github.com.../testify/suite/suite.go:202&#xA;testing.tRunner&#xA;&#x9;..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/testing/testing.go:1595&#xA;2024-09-10T16:47:49.179Z&#x9;DEBUG&#x9;storenodes/storenodes.go:98&#x9;loaded communities storenodes&#x9;{&#34;CommunityStorenodes&#34;: {&#34;count&#34;: 0}}&#xA;2024-09-10T16:47:49.180Z&#x9;DEBUG&#x9;hasPairedDevices&#x9;protocol/messenger.go:2097&#x9;installations info&#x9;{&#34;Number of installations&#34;: 1, &#34;Number of enabled installations&#34;: 1}&#xA;2024-09-10T16:47:49.180Z&#x9;DEBUG&#x9;hasPairedDevices&#x9;protocol/messenger.go:2097&#x9;installations info&#x9;{&#34;Number of installations&#34;: 1, &#34;Number of enabled installations&#34;: 1}&#xA;2024-09-10T16:47:49.202Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;049ee811&#34;, &#34;epoch&#34;: 1561}}&#xA;2024-09-10T16:47:49.202Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;041caacc&#34;, &#34;epoch&#34;: 858}}&#xA;2024-09-10T16:47:49.202Z&#x9;DEBUG&#x9;messenger-Alice&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0404ab81&#34;, &#34;epoch&#34;: 850}}&#xA;2024-09-10T16:47:49.204Z&#x9;DEBUG&#x9;transport/filters_manager.go:612&#x9;registering filter for&#x9;{&#34;filtersManager&#34;: {&#34;chatID&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83-contact-code&#34;, &#34;type&#34;: &#34;contact-code&#34;, &#34;topic&#34;: &#34;0xa7171021&#34;}}&#xA;2024-09-10T16:47:49.204Z&#x9;DEBUG&#x9;protocol/messenger.go:5486&#x9;sending private message&#x9;{&#34;site&#34;: &#34;Send&#34;, &#34;chatID&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;}&#xA;2024-09-10T16:47:49.204Z&#x9;DEBUG&#x9;common/message_sender.go:164&#x9;sending a private message&#x9;{&#34;public-key&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;, &#34;site&#34;: &#34;SendPrivate&#34;}&#xA;2024-09-10T16:47:49.204Z&#x9;DEBUG&#x9;common/message_sender.go:446&#x9;sending private message&#x9;{&#34;recipient&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;}&#xA;2024-09-10T16:47:49.205Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:724&#x9;handling message scheduled&#xA;2024-09-10T16:47:49.205Z&#x9;DEBUG&#x9;node/node.go:293&#x9;Sending message&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0468d3e5&#34;, &#34;groupID&#34;: &#34;aa5832b8&#34;, &#34;id&#34;: &#34;6255625a&#34;}}&#xA;2024-09-10T16:47:49.205Z&#x9;DEBUG&#x9;hasPairedDevices&#x9;protocol/messenger.go:2097&#x9;installations info&#x9;{&#34;Number of installations&#34;: 1, &#34;Number of enabled installations&#34;: 1}&#xA;2024-09-10T16:47:49.206Z&#x9;DEBUG&#x9;protocol/messenger.go:2504&#x9;inside sendChatMessage&#x9;{&#34;id&#34;: &#34;0x0de4cea75abd11c4c0bfa0bb8a5d2acbca755e8a970c6b1669daedec7c9aafda&#34;, &#34;from&#34;: &#34;0x0468d3e5684b73cabf155749ce52aa9e788945bb26f98dfbd9a1d38bdf8217074ea0a2b16a4de4da4549346f3ba9544863deae1b82cfb18adf49a2946f7d8cba53&#34;, &#34;displayName&#34;: &#34;&#34;, &#34;ChatId&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;, &#34;Clock&#34;: &#34;1725986869181&#34;, &#34;Timestamp&#34;: &#34;1725986869181&#34;}&#xA;2024-09-10T16:47:49.207Z&#x9;DEBUG&#x9;protocol/messenger.go:5486&#x9;sending private message&#x9;{&#34;site&#34;: &#34;Send&#34;, &#34;chatID&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;}&#xA;2024-09-10T16:47:49.207Z&#x9;DEBUG&#x9;common/message_sender.go:164&#x9;sending a private message&#x9;{&#34;public-key&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;, &#34;site&#34;: &#34;SendPrivate&#34;}&#xA;2024-09-10T16:47:49.207Z&#x9;DEBUG&#x9;common/message_sender.go:446&#x9;sending private message&#x9;{&#34;recipient&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;}&#xA;2024-09-10T16:47:49.207Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:724&#x9;handling message scheduled&#xA;2024-09-10T16:47:49.207Z&#x9;DEBUG&#x9;node/node.go:293&#x9;Sending message&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0468d3e5&#34;, &#34;groupID&#34;: &#34;aa5832b8&#34;, &#34;id&#34;: &#34;88f3fb92&#34;}}&#xA;2024-09-10T16:47:49.207Z&#x9;DEBUG&#x9;hasPairedDevices&#x9;protocol/messenger.go:2097&#x9;installations info&#x9;{&#34;Number of installations&#34;: 1, &#34;Number of enabled installations&#34;: 1}&#xA;2024-09-10T16:47:49.208Z&#x9;DEBUG&#x9;protocol/messenger.go:2504&#x9;inside sendChatMessage&#x9;{&#34;id&#34;: &#34;0x9367f65be99197dc472e6fbe40e2719f3f74c13494a2fcff5423aad942d0ff57&#34;, &#34;from&#34;: &#34;0x0468d3e5684b73cabf155749ce52aa9e788945bb26f98dfbd9a1d38bdf8217074ea0a2b16a4de4da4549346f3ba9544863deae1b82cfb18adf49a2946f7d8cba53&#34;, &#34;displayName&#34;: &#34;&#34;, &#34;ChatId&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;, &#34;Clock&#34;: &#34;1725986869206&#34;, &#34;Timestamp&#34;: &#34;1725986869206&#34;}&#xA;2024-09-10T16:47:49.209Z&#x9;DEBUG&#x9;protocol/messenger.go:5486&#x9;sending private message&#x9;{&#34;site&#34;: &#34;Send&#34;, &#34;chatID&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;}&#xA;2024-09-10T16:47:49.209Z&#x9;DEBUG&#x9;common/message_sender.go:164&#x9;sending a private message&#x9;{&#34;public-key&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;, &#34;site&#34;: &#34;SendPrivate&#34;}&#xA;2024-09-10T16:47:49.209Z&#x9;DEBUG&#x9;common/message_sender.go:446&#x9;sending private message&#x9;{&#34;recipient&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;}&#xA;2024-09-10T16:47:49.209Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:724&#x9;handling message scheduled&#xA;2024-09-10T16:47:49.209Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0468d3e5&#34;, &#34;epoch&#34;: 2}}&#xA;2024-09-10T16:47:49.209Z&#x9;DEBUG&#x9;node/node.go:293&#x9;Sending message&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0468d3e5&#34;, &#34;groupID&#34;: &#34;aa5832b8&#34;, &#34;id&#34;: &#34;7fc5dfc3&#34;}}&#xA;2024-09-10T16:47:49.209Z&#x9;DEBUG&#x9;hasPairedDevices&#x9;protocol/messenger.go:2097&#x9;installations info&#x9;{&#34;Number of installations&#34;: 1, &#34;Number of enabled installations&#34;: 1}&#xA;2024-09-10T16:47:49.210Z&#x9;DEBUG&#x9;protocol/messenger.go:2504&#x9;inside sendChatMessage&#x9;{&#34;id&#34;: &#34;0xf68e8c379bbbd8da0fa9092f3d6ec6e04efe1e3a2627ffaf0cf36cb1d878ff34&#34;, &#34;from&#34;: &#34;0x0468d3e5684b73cabf155749ce52aa9e788945bb26f98dfbd9a1d38bdf8217074ea0a2b16a4de4da4549346f3ba9544863deae1b82cfb18adf49a2946f7d8cba53&#34;, &#34;displayName&#34;: &#34;&#34;, &#34;ChatId&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;, &#34;Clock&#34;: &#34;1725986869208&#34;, &#34;Timestamp&#34;: &#34;1725986869208&#34;}&#xA;2024-09-10T16:47:49.256Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04485dfe&#34;, &#34;epoch&#34;: 1955}}&#xA;2024-09-10T16:47:49.258Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;048b92e4&#34;, &#34;epoch&#34;: 2224}}&#xA;2024-09-10T16:47:49.272Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04ad37f5&#34;, &#34;epoch&#34;: 1934}}&#xA;2024-09-10T16:47:49.272Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040b3c22&#34;, &#34;epoch&#34;: 1558}}&#xA;2024-09-10T16:47:49.277Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040cd6d3&#34;, &#34;epoch&#34;: 1551}}&#xA;2024-09-10T16:47:49.340Z&#x9;DEBUG&#x9;messenger-Bobby&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040a5dd5&#34;, &#34;epoch&#34;: 849}}&#xA;2024-09-10T16:47:49.422Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04189afe&#34;, &#34;epoch&#34;: 2171}}&#xA;2024-09-10T16:47:49.423Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04420f87&#34;, &#34;epoch&#34;: 2215}}&#xA;2024-09-10T16:47:49.423Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04f955ef&#34;, &#34;epoch&#34;: 2195}}&#xA;2024-09-10T16:47:49.423Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0458c277&#34;, &#34;epoch&#34;: 2205}}&#xA;2024-09-10T16:47:49.423Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0489298c&#34;, &#34;epoch&#34;: 1077}}&#xA;2024-09-10T16:47:49.444Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;045add39&#34;, &#34;epoch&#34;: 273}}&#xA;2024-09-10T16:47:49.459Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04a90e8f&#34;, &#34;epoch&#34;: 1555}}&#xA;2024-09-10T16:47:49.476Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04950fc7&#34;, &#34;epoch&#34;: 1}}&#xA;2024-09-10T16:47:49.503Z&#x9;DEBUG&#x9;messenger-Alice&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0404ab81&#34;, &#34;epoch&#34;: 851}}&#xA;2024-09-10T16:47:49.503Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;049ee811&#34;, &#34;epoch&#34;: 1562}}&#xA;2024-09-10T16:47:49.503Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;041caacc&#34;, &#34;epoch&#34;: 859}}&#xA;2024-09-10T16:47:49.509Z&#x9;DEBUG&#x9;node/node.go:373&#x9;sending MESSAGE&#x9;{&#34;mvds&#34;: {&#34;groupID&#34;: &#34;aa5832b8&#34;, &#34;from&#34;: &#34;0468d3e5&#34;, &#34;to&#34;: &#34;04950fc7&#34;, &#34;messageID&#34;: &#34;6255625a&#34;}}&#xA;2024-09-10T16:47:49.509Z&#x9;DEBUG&#x9;node/node.go:373&#x9;sending MESSAGE&#x9;{&#34;mvds&#34;: {&#34;groupID&#34;: &#34;aa5832b8&#34;, &#34;from&#34;: &#34;0468d3e5&#34;, &#34;to&#34;: &#34;04950fc7&#34;, &#34;messageID&#34;: &#34;88f3fb92&#34;}}&#xA;2024-09-10T16:47:49.510Z&#x9;DEBUG&#x9;encryption/encryptor.go:509&#x9;no installations, sending to all devices&#x9;{&#34;encryptor&#34;: {&#34;site&#34;: &#34;EncryptPayload&#34;, &#34;their-identity-key&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;}}&#xA;2024-09-10T16:47:49.533Z&#x9;DEBUG&#x9;transport/filters_manager.go:477&#x9;registering filter for&#x9;{&#34;filtersManager&#34;: {&#34;chatID&#34;: &#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83-negotiated&#34;, &#34;type&#34;: &#34;negotiated&#34;, &#34;topic&#34;: &#34;0x1752a952&#34;}}&#xA;2024-09-10T16:47:49.533Z&#x9;DEBUG&#x9;common/message_segmentation.go:47&#x9;message segmented&#x9;{&#34;segments&#34;: 1}&#xA;2024-09-10T16:47:49.533Z&#x9;DEBUG&#x9;common/message_sender.go:1227&#x9;sending partitioned topic&#x9;{&#34;site&#34;: &#34;sendMessageSpec&#34;}&#xA;2024-09-10T16:47:49.536Z&#x9;DEBUG&#x9;transport/filters_manager.go:441&#x9;registering filter for&#x9;{&#34;filtersManager&#34;: {&#34;chatID&#34;: &#34;contact-discovery-872&#34;, &#34;type&#34;: &#34;partitioned&#34;, &#34;topic&#34;: &#34;0x19c723b4&#34;}}&#xA;2024-09-10T16:47:49.536Z&#x9;DEBUG&#x9;waku/waku.go:999&#x9;send: sending envelope&#x9;{&#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;}&#xA;2024-09-10T16:47:49.536Z&#x9;DEBUG&#x9;waku/waku.go:1275&#x9;addAndBridge: processing envelope&#x9;{&#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;, &#34;site&#34;: &#34;addAndBridge&#34;, &#34;topic&#34;: &#34;0x19c723b4&#34;, &#34;isP2P&#34;: false}&#xA;2024-09-10T16:47:49.536Z&#x9;DEBUG&#x9;waku/waku.go:1330&#x9;addAndBridge: adding envelope&#x9;{&#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;, &#34;site&#34;: &#34;addAndBridge&#34;, &#34;topic&#34;: &#34;0x19c723b4&#34;, &#34;isP2P&#34;: false}&#xA;2024-09-10T16:47:49.536Z&#x9;DEBUG&#x9;protocol/messenger_peersyncing.go:415&#x9;sent private messages&#x9;{&#34;messageIDs&#34;: [&#34;0x0de4cea75abd11c4c0bfa0bb8a5d2acbca755e8a970c6b1669daedec7c9aafda&#34;, &#34;0x9367f65be99197dc472e6fbe40e2719f3f74c13494a2fcff5423aad942d0ff57&#34;], &#34;hashes&#34;: [&#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;]}&#xA;2024-09-10T16:47:49.536Z&#x9;DEBUG&#x9;pushnotificationclient/client.go:729&#x9;handling message sent&#xA;2024-09-10T16:47:49.536Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;0468d3e5&#34;, &#34;epoch&#34;: 3}}&#xA;2024-09-10T16:47:49.557Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04485dfe&#34;, &#34;epoch&#34;: 1956}}&#xA;2024-09-10T16:47:49.558Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;048b92e4&#34;, &#34;epoch&#34;: 2225}}&#xA;2024-09-10T16:47:49.573Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;04ad37f5&#34;, &#34;epoch&#34;: 1935}}&#xA;2024-09-10T16:47:49.573Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040b3c22&#34;, &#34;epoch&#34;: 1559}}&#xA;2024-09-10T16:47:49.577Z&#x9;DEBUG&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040cd6d3&#34;, &#34;epoch&#34;: 1552}}&#xA;2024-09-10T16:47:49.594Z&#x9;DEBUG&#x9;protocol/messenger_status_updates.go:104&#x9;Error cleaning status updates&#x9;{&#34;error&#34;: &#34;sql: database is closed&#34;}&#xA;2024-09-10T16:47:49.601Z&#x9;DEBUG&#x9;waku/waku.go:1457&#x9;expiring envelope&#x9;{&#34;site&#34;: &#34;expire&#34;, &#34;hash&#34;: &#34;0xcd268769d6d2dbfbb7f9745e1e44c5eebf0912fb369f399f3380b1f0d0005e04&#34;}&#xA;2024-09-10T16:47:49.624Z&#x9;DEBUG&#x9;waku/api.go:501&#x9;retrieved filter message&#x9;{&#34;site&#34;: &#34;getFilterMessages&#34;, &#34;filterId&#34;: &#34;d85c5187656fc6cd352f0986dfecfbcdb564474fa877967028290f531b911072&#34;, &#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;, &#34;isP2P&#34;: false, &#34;topic&#34;: &#34;0x19c723b4&#34;}&#xA;2024-09-10T16:47:49.624Z&#x9;DEBUG&#x9;transport/transport.go:263&#x9;message not cached&#x9;{&#34;Transport&#34;: {&#34;site&#34;: &#34;retrieveRawAll&#34;, &#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;}}&#xA;2024-09-10T16:47:49.624Z&#x9;DEBUG&#x9;common/message_segmentation.go:237&#x9;handling message segment&#x9;{&#34;site&#34;: &#34;handleSegmentationLayerV2&#34;, &#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;, &#34;EntireMessageHash&#34;: &#34;0x&#34;, &#34;Index&#34;: 0, &#34;SegmentsCount&#34;: 0, &#34;ParitySegmentIndex&#34;: 0, &#34;ParitySegmentsCount&#34;: 0}&#xA;2024-09-10T16:47:49.624Z&#x9;DEBUG&#x9;common/message_sender.go:969&#x9;failed to handle segmentation layer message&#x9;{&#34;site&#34;: &#34;handleMessage&#34;, &#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;, &#34;error&#34;: &#34;invalid segments count&#34;, &#34;errorVerbose&#34;: &#34;invalid segments count\ngithub..../status-go/protocol/common.init\n\.../protocol/common/message_segmentation.go:22\nruntime.doInit1\n\t..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/runtime/proc.go:6735\nruntime.doInit\n\t..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/runtime/proc.go:6702\nruntime.main\n\t..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/runtime/proc.go:249\nruntime.goexit\n\t..................................../nix/store/vxxlsbznhhkw1sb7qi5wki03d2sihpha-go-1.21..../src/runtime/asm_amd64.s:1650&#34;}&#xA;2024-09-10T16:47:49.624Z&#x9;DEBUG&#x9;encryption/protocol.go:756&#x9;received a protocol message&#x9;{&#34;Protocol&#34;: {&#34;site&#34;: &#34;HandleMessage&#34;, &#34;sender-public-key&#34;: &#34;0x0468d3e5684b73cabf155749ce52aa9e788945bb26f98dfbd9a1d38bdf8217074ea0a2b16a4de4da4549346f3ba9544863deae1b82cfb18adf49a2946f7d8cba53&#34;, &#34;my-installation-id&#34;: &#34;f713136a-063c-4aa1-b7fc-ef972cb7d08f&#34;, &#34;messageID&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;}}&#xA;2024-09-10T16:47:49.625Z&#x9;DEBUG&#x9;datasync/datasync.go:41&#x9;handling datasync message&#x9;{&#34;site&#34;: &#34;Handle&#34;}&#xA;2024-09-10T16:47:49.625Z&#x9;DEBUG&#x9;node/node.go:528&#x9;MESSAGE received&#x9;{&#34;mvds&#34;: {&#34;from&#34;: &#34;0468d3e5&#34;, &#34;to&#34;: &#34;04950fc7&#34;, &#34;messageID&#34;: &#34;6255625a&#34;}}&#xA;2024-09-10T16:47:49.625Z&#x9;DEBUG&#x9;node/node.go:512&#x9;sending ACK&#x9;{&#34;mvds&#34;: {&#34;groupID&#34;: &#34;aa5832b8&#34;, &#34;from&#34;: &#34;04950fc7&#34;, &#34;&#34;: &#34;0468d3e5&#34;, &#34;messageID&#34;: &#34;6255625a&#34;}}&#xA;2024-09-10T16:47:49.625Z&#x9;DEBUG&#x9;node/node.go:528&#x9;MESSAGE received&#x9;{&#34;mvds&#34;: {&#34;from&#34;: &#34;0468d3e5&#34;, &#34;to&#34;: &#34;04950fc7&#34;, &#34;messageID&#34;: &#34;88f3fb92&#34;}}&#xA;2024-09-10T16:47:49.625Z&#x9;DEBUG&#x9;node/node.go:512&#x9;sending ACK&#x9;{&#34;mvds&#34;: {&#34;groupID&#34;: &#34;aa5832b8&#34;, &#34;from&#34;: &#34;04950fc7&#34;, &#34;&#34;: &#34;0468d3e5&#34;, &#34;messageID&#34;: &#34;88f3fb92&#34;}}&#xA;2024-09-10T16:47:49.625Z&#x9;DEBUG&#x9;protocol/messenger.go:3557&#x9;processing messages further&#x9;{&#34;site&#34;: &#34;RetrieveAll&#34;, &#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;, &#34;count&#34;: 2}&#xA;2024-09-10T16:47:49.642Z&#x9;DEBUG&#x9;messenger-Bobby&#x9;node/node.go:221&#x9;Epoch processing&#x9;{&#34;mvds&#34;: {&#34;node&#34;: &#34;040a5dd5&#34;, &#34;epoch&#34;: 850}}&#xA;2024-09-10T16:47:49.648Z&#x9;DEBUG&#x9;transport/filters_manager.go:477&#x9;registering filter for&#x9;{&#34;filtersManager&#34;: {&#34;chatID&#34;: &#34;0x0468d3e5684b73cabf155749ce52aa9e788945bb26f98dfbd9a1d38bdf8217074ea0a2b16a4de4da4549346f3ba9544863deae1b82cfb18adf49a2946f7d8cba53-negotiated&#34;, &#34;type&#34;: &#34;negotiated&#34;, &#34;topic&#34;: &#34;0x1752a952&#34;}}&#xA;2024-09-10T16:47:49.648Z&#x9;INFO&#x9;protocol/messenger.go:3573&#x9;processing message&#x9;{&#34;site&#34;: &#34;RetrieveAll&#34;, &#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;, &#34;message-id&#34;: &#34;0x0de4cea75abd11c4c0bfa0bb8a5d2acbca755e8a970c6b1669daedec7c9aafda&#34;, &#34;type&#34;: &#34;CHAT_MESSAGE&#34;, &#34;senderID&#34;: &#34;0x0468d3e5684b73cabf155749ce52aa9e788945bb26f98dfbd9a1d38bdf8217074ea0a2b16a4de4da4549346f3ba9544863deae1b82cfb18adf49a2946f7d8cba53&#34;}&#xA;2024-09-10T16:47:49.649Z&#x9;INFO&#x9;protocol/messenger_handlers.go:277&#x9;handling ChatMessage&#xA;2024-09-10T16:47:49.650Z&#x9;DEBUG&#x9;protocol/messenger.go:3631&#x9;Handled parsed message&#x9;{&#34;site&#34;: &#34;RetrieveAll&#34;, &#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;, &#34;message-id&#34;: &#34;0x0de4cea75abd11c4c0bfa0bb8a5d2acbca755e8a970c6b1669daedec7c9aafda&#34;}&#xA;2024-09-10T16:47:49.650Z&#x9;INFO&#x9;protocol/messenger.go:3573&#x9;processing message&#x9;{&#34;site&#34;: &#34;RetrieveAll&#34;, &#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;, &#34;message-id&#34;: &#34;0x9367f65be99197dc472e6fbe40e2719f3f74c13494a2fcff5423aad942d0ff57&#34;, &#34;type&#34;: &#34;CHAT_MESSAGE&#34;, &#34;senderID&#34;: &#34;0x0468d3e5684b73cabf155749ce52aa9e788945bb26f98dfbd9a1d38bdf8217074ea0a2b16a4de4da4549346f3ba9544863deae1b82cfb18adf49a2946f7d8cba53&#34;}&#xA;2024-09-10T16:47:49.650Z&#x9;INFO&#x9;protocol/messenger_handlers.go:277&#x9;handling ChatMessage&#xA;2024-09-10T16:47:49.651Z&#x9;DEBUG&#x9;protocol/messenger.go:3631&#x9;Handled parsed message&#x9;{&#34;site&#34;: &#34;RetrieveAll&#34;, &#34;hash&#34;: &#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;, &#34;message-id&#34;: &#34;0x9367f65be99197dc472e6fbe40e2719f3f74c13494a2fcff5423aad942d0ff57&#34;}&#xA;2024-09-10T16:47:49.651Z&#x9;DEBUG&#x9;transport/transport.go:547&#x9;confirming message processed&#x9;{&#34;Transport&#34;: {&#34;ids&#34;: [&#34;0x08197db3a63caa5dad1dc997d27a6bc5cc2b9c57a11493e1294ee59b9803b33b&#34;], &#34;timestamp&#34;: 1725986869651}}&#xA;    messenger_send_images_album_test.go:250: &#xA;        &#x9;Error Trace:&#x9;.../status-go_prs_tests_PR-5823/protocol/messenger_send_images_album_test.go:250&#xA;        &#x9;Error:      &#x9;&#34;[clock:1725986869206  timestamp:1725986869206  text:&#34;You can edit me now&#34;  chat_id:&#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;  message_type:ONE_TO_ONE  content_type:IMAGE  image:{payload:&#34;\xff\xd8\xff\xdb\x00C\x00\x03\x02\x02\x02\x02\x02\x03\x02\x02\x02\x03\x03\x03\x03\x04\x06\x04\x04\x04\x04\x04\x08\x06\x06\x05\x06\t\x08\n\n\t\x08\t\t\n\x0c\x0f\x0c\n\x0b\x0e\x0b\t\t\r\x11\r\x0e\x0f\x10\x10\x11\x10\n\x0c\x12\x13\x12\x10\x13\x0f\x10\x10\x10\xff\xc9\x00\x0b\x08\x00\x01\x00\x01\x01\x01\x11\x00\xff\xcc\x00\x06\x00\x10\x10\x05\xff\xda\x00\x08\x01\x01\x00\x00?\x00\xd2\xcf \xff\xd9&#34;  format:JPEG  album_id:&#34;e3628bdf-bf3e-469c-a4b0-264bfc56378a&#34;  width:1200  height:1000  album_images_count:3} clock:1725986869181  timestamp:1725986869181  text:&#34;You can edit me now&#34;  chat_id:&#34;0x04950fc7a3b1ff7317c5c1570052d7c8608be760b38942cab545ea3d2f73a384b80fba796251d3410b69f824e11fe59bdbe718cb316cbf18f5fb5f72e1e68a2b83&#34;  message_type:ONE_TO_ONE  content_type:IMAGE  image:{payload:&#34;\xff\xd8\xff\xdb\x00C\x00\x03\x02\x02\x02\x02\x02\x03\x02\x02\x02\x03\x03\x03\x03\x04\x06\x04\x04\x04\x04\x04\x08\x06\x06\x05\x06\t\x08\n\n\t\x08\t\t\n\x0c\x0f\x0c\n\x0b\x0e\x0b\t\t\r\x11\r\x0e\x0f\x10\x10\x11\x10\n\x0c\x12\x13\x12\x10\x13\x0f\x10\x10\x10\xff\xc9\x00\x0b\x08\x00\x01\x00\x01\x01\x01\x11\x00\xff\xcc\x00\x06\x00\x10\x10\x05\xff\xda\x00\x08\x01\x01\x00\x00?\x00\xd2\xcf \xff\xd9&#34;  format:JPEG  album_id:&#34;e3628bdf-bf3e-469c-a4b0-264bfc56378a&#34;  width:1200  height:1000  album_images_count:3}]&#34; should have 3 item(s), but has 2&#xA;        &#x9;Test:       &#x9;TestMessengerSendImagesAlbumSuite/TestAlbumImageEditText&#xA;2024-09-10T16:47:49.652Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 0}&#xA;2024-09-10T16:47:49.652Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 1}&#xA;2024-09-10T16:47:49.652Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 2}&#xA;2024-09-10T16:47:49.652Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 3}&#xA;2024-09-10T16:47:49.652Z&#x9;DEBUG&#x9;CommunityRekeyLoop&#x9;protocol/messenger_communities.go:4671&#x9;CommunityRekeyLoop stopped&#xA;2024-09-10T16:47:49.652Z&#x9;DEBUG&#x9;PeerSyncingLoop&#x9;protocol/messenger_peersyncing.go:110&#x9;peersyncing loop stopped&#xA;2024-09-10T16:47:49.652Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 4}&#xA;2024-09-10T16:47:49.652Z&#x9;DEBUG&#x9;communities/manager.go:541&#x9;quitting ens verification loop&#xA;2024-09-10T16:47:49.652Z&#x9;DEBUG&#x9;communities/manager.go:621&#x9;quitting owner verification loop&#xA;2024-09-10T16:47:49.652Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 5}&#xA;2024-09-10T16:47:49.653Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 6}&#xA;2024-09-10T16:47:49.653Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 7}&#xA;2024-09-10T16:47:49.653Z&#x9;INFO&#x9;node/node.go:245&#x9;Stopping node&#x9;{&#34;mvds&#34;: {}}&#xA;2024-09-10T16:47:49.653Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 8}&#xA;2024-09-10T16:47:49.653Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 9}&#xA;2024-09-10T16:47:49.653Z&#x9;DEBUG&#x9;protocol/messenger.go:1481&#x9;quitting encryption subscription loop&#xA;2024-09-10T16:47:49.653Z&#x9;INFO&#x9;node/node.go:203&#x9;reset data sync for peer stopped&#x9;{&#34;mvds&#34;: {}}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 0}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 1}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 2}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 3}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 4}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 5}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 6}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;communities/manager.go:541&#x9;quitting ens verification loop&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 7}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;PeerSyncingLoop&#x9;protocol/messenger_peersyncing.go:110&#x9;peersyncing loop stopped&#xA;2024-09-10T16:47:49.654Z&#x9;INFO&#x9;node/node.go:245&#x9;Stopping node&#x9;{&#34;mvds&#34;: {}}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;communities/manager.go:621&#x9;quitting owner verification loop&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 8}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;CommunityRekeyLoop&#x9;protocol/messenger_communities.go:4671&#x9;CommunityRekeyLoop stopped&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;protocol/messenger.go:1999&#x9;running shutdown task&#x9;{&#34;n&#34;: 9}&#xA;2024-09-10T16:47:49.654Z&#x9;DEBUG&#x9;protocol/messenger.go:1481&#x9;quitting encryption subscription loop&#xA;2024-09-10T16:47:49.654Z&#x9;INFO&#x9;node/node.go:203&#x9;reset data sync for peer stopped&#x9;{&#34;mvds&#34;: {}}&#xA;--- FAIL: TestMessengerSendImagesAlbumSuite/TestAlbumImageEditText (1.66s)&#xA;
> > ```
> 
> </details>


> 
> ```
> github.com/status-im/status-go/protocol TestMessengerSendImagesAlbumSuite
> ```
> 
> <details><summary>Stack Traces | 7.86s run time</summary>
> 
> > 
> > ```
> > === RUN   TestMessengerSendImagesAlbumSuite&#xA;--- FAIL: TestMessengerSendImagesAlbumSuite (7.86s)&#xA;
> > ```
> 
> </details>

</details>

To view individual test run time comparison to the main branch, go to the [Test Analytics Dashboard](https://app.codecov.io/gh/status-im/status-go/tests/fix%2Ftag-version)

--- a/.github/workflows/commit-check.yml
+++ b/.github/workflows/commit-check.yml
@@ -32,12 +32,13 @@ jobs:
           echo "exit_code=$exit_code" >> $GITHUB_OUTPUT
           
           if [[ $exit_code -ne 0 ]]; then
+            invalid_commit_messages=$(echo $output | sed '1d;$d')
             EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
             echo "error_message<<$EOF" >> "$GITHUB_ENV"
-            echo "$output" >> "$GITHUB_ENV"
+            echo "invalid_commit_messages" >> "$GITHUB_ENV"
             echo "$EOF" >> "$GITHUB_ENV"
           else
-            has_breaking_changes=$(echo "$output" | sed -n '2p')
+            has_breaking_changes=$(echo "$output" | tail -n 1)
             echo "has_breaking_changes=$has_breaking_changes" >> $GITHUB_OUTPUT
           fi
 

--- a/_assets/scripts/commit_check.sh
+++ b/_assets/scripts/commit_check.sh
@@ -1,36 +1,4 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
-
-parse_commits() {
-
-    BASE_BRANCH=${BASE_BRANCH:-develop}
-
-    start_commit=${1:-origin/${BASE_BRANCH}}
-    end_commit=${2:-HEAD}
-    is_breaking_change=false
-    exit_code=0
-
-    echo "checking commits between: $start_commit $end_commit"
-    # Run the loop in the current shell using process substitution
-    while IFS= read -r message || [ -n "$message" ]; do
-        # Check if commit message follows conventional commits format
-        if [[ $message =~ ^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\(.*\))?(\_|!):.*$ ]]; then
-            # Check for breaking changes
-            if [[ ${BASH_REMATCH[3]} == *'!'* ]]; then
-                is_breaking_change=true
-            fi
-        else
-            echo "Commit message \"$message\" is not well-formed"
-            exit_code=1
-        fi
-    done < <(git log --format=%s "$start_commit".."$end_commit")
-
-    if [[ $exit_code -ne 0 ]]; then
-        exit ${exit_code}
-    fi
-
-    echo "$is_breaking_change"
-}
-
+source _assets/scripts/parse_commits.sh
 parse_commits "$@"

--- a/_assets/scripts/parse_commits.sh
+++ b/_assets/scripts/parse_commits.sh
@@ -1,0 +1,38 @@
+#!/usr/bin/env bash
+
+# The output of this script is as follows:
+# 1. One line "checking commits between: <start_commit> <end_commit>"
+# 2. One line for each commit message that is not well-formed
+# 3. One line with the value of "is_breaking_change" (true/false)
+
+set -euo pipefail
+
+source _assets/scripts/colors.sh
+
+parse_commits() {
+
+    BASE_BRANCH=${BASE_BRANCH:-develop}
+
+    start_commit=${1:-origin/${BASE_BRANCH}}
+    end_commit=${2:-HEAD}
+    is_breaking_change=false
+    exit_code=0
+
+    echo -e "${GRN}Checking commits between:${RST} $start_commit $end_commit"
+    # Run the loop in the current shell using process substitution
+    while IFS= read -r message || [ -n "$message" ]; do
+        # Check if commit message follows conventional commits format
+        if [[ $message =~ ^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\(.*\))?(\_|!):.*$ ]]; then
+            # Check for breaking changes
+            if [[ ${BASH_REMATCH[3]} == *'!'* ]]; then
+                is_breaking_change=true
+            fi
+        else
+            echo -e "${YLW}Commit message is not well-formed:${RST} \"$message\""
+            exit_code=1
+        fi
+    done < <(git log --format=%s "$start_commit".."$end_commit")
+
+    echo "$is_breaking_change"
+    exit ${exit_code}
+}

--- a/_assets/scripts/tag_version.sh
+++ b/_assets/scripts/tag_version.sh
@@ -2,7 +2,8 @@
 
 set -euo pipefail
 
-source _assets/scripts/commit_check.sh
+source _assets/scripts/parse_commits.sh
+source _assets/scripts/colors.sh
 
 get_latest_tag() {
     # Get the latest tag on develop
@@ -28,26 +29,44 @@ bump_version() {
 }
 
 calculate_new_version() {
-    # Get the latest tag
-    latest_tag=$(get_latest_tag)
-
-    echo "calculating new tag from $latest_tag and $1" >&2
+    target_commit=$1
+    latest_tag=$2
 
     # Parse commits to determine if there are breaking changes
-    is_breaking_change=$(parse_commits "$latest_tag" "$1")
+    output=$(parse_commits "$latest_tag" "$target_commit")
+    exit_code=$?
+
+    a=$(echo "$output" | sed '$d')
+    is_breaking_change=$(echo "$output" | tail -n 1)
+
+    echo "$a" >&2
+
+
+    if [[ $is_breaking_change == 'true' ]]; then
+      echo -e "${YLW}Breaking change detected${RST}" >&2
+    fi
+
+    if [[ $exit_code -ne 0 && $is_breaking_change != true ]]; then
+        echo -e "${YLW}Some commits are ill-formed, can not to auto-calculate new version${RST}" >&2
+        read -p "Any of the commits above have a breaking change? (y/n): " yn
+        case $yn in
+            [Yy]* ) is_breaking_change=true;;
+            [Nn]* ) is_breaking_change=false;;
+            * ) echo "Please answer yes or no."; exit 1;;
+        esac
+    fi
 
     # Bump version accordingly
-    echo "$(bump_version "$latest_tag" "$is_breaking_change")"
-  }
-
-
-main() {
-    new_version=$(calculate_new_version "$1")
-    echo "calculated new version: $new_version" >&2
-
-    git tag -a "$new_version" "$1" -m "release $new_version"
+    bump_version "$latest_tag" "$is_breaking_change"
 }
 
-target_commit=${1:-HEAD}
+latest_tag=$(get_latest_tag)
+echo -e "${GRN}Latest tag found:${RST} $latest_tag" >&2
 
-main "$target_commit"
+target_commit=${1:-HEAD}
+echo -e "${GRN}Calculating new version for:${RST} $target_commit" >&2
+
+new_version=$(calculate_new_version "$target_commit" "$latest_tag")
+echo -e "${GRN}Calculated new version:${RST} $new_version" >&2
+
+git tag -a "$new_version" "$target_commit" -m "release $new_version"

